rchain/crypto.py

```diff
@@ -71,8 +80,18 @@ def sign(self, data: bytes) -> bytes:
             data, hashfunc=blake2b_32, sigencode=sigencode_der_canonize
         )
 
+    def sign_digest(self, digest: bytes) -> bytes:
+        return self._key.sign_digest(digest, sigencode=sigencode_der_canonize)
```

Comment:
Just curious, who/what is going to use this? This class abstract away the algorithms, but user of this method has to assume correct hash algorithm used for RChain validator keys.

Reply:
signing the block hash would be useful because the block hash is after-hash.

Reply:
Got it. Can you name it `sign_block_hash` then?

Reply:
I don't have any other cases. Then `sign_block_hash` it is.